content/en/docs/use-cases/bootstrapping/index.md

```diff
@@ -82,6 +84,8 @@ This will cause the following to happen:
 
 - If the user selects the "Passkey from another device" option (NOTE: the exact text will vary slightly by platform), then the browser/platform will guide the user through using a FIDO2 security key or the  Cross-Device Authentication (CDA) flow to use a passkey from their smartphone or tablet to deliver a WebAuthn response to the `navigator.credentials.get()` call.
 
+- Send the WebAuthn response to your server for verification and additional security checks.
```

Comment:
I don't think this line goes far enough. "...and then do what?" It's implied the user should get logged in, but why not explicitly mention that for sake of thoroughness?